### DISCUSSION TITLE
The geoip fields are empty after Upgrade 2.3.200 [20230113]

### DISCUSSION THREAD

**Discussion by sleepingbel** [Security-Onion-Solutions/securityonion#9609] [Unsupported Versions] [answered]
Hello all,

initial install 2.3.140 Hotfix [20220812], upgraded whit every new version until version Upgrade 2.3.200 [20230113]

Virtual machine on ESXI 6.7
Standalone install, zeek network logs
8 vCPU
24 vRAM
1.5 TB diskspace 

After the upgrade to 2.3.200 all the geoip fields are empty.

1. I have whitelisted the following domains on my external firewall:
         - geoip.elastic.co (GeoIP updates for Elasticsearch)
        - storage.googleapis.com (GeoIP updates for Elasticsearch)
2. I have performed what is in the documentation [Elasticsearch GeoIP ](https://docs.securityonion.net/en/2.3/elasticsearch.html?highlight=geoip#geoip)
3. I rebooted SO

How can I check if the geoip enrichment data is present in SO?
How can I check if the geoip data is updated?

Regards
Bart

**Comment by InfosecGoon** [comment, marked as answer]
Is there anything helpful in the Elasticsearch logs, if you grep for 'GeoLite' in /opt/so/log/elasticsearch/*log?

**Reply by sleepingbel** [reply]
The problem hase solved itself. I did not find any errors regarding geolite in the logs.
Stil thanks for the help.
